### PR TITLE
Update project

### LINKED ISSUE #27: Provide a way to detect license directly from PyPI's metadata
**Is your feature request related to a problem? Please describe.**

As a user of Thoth's license solver, I would like to have the ability to detect licenses directly from PyPI, something like this could do the trick: 

```
curl https://pypi.org/pypi/requests/json | thoth-license-solver --pypi-json
```

Alternatively (maybe more user friendly), a user can specify package and version and the tool will download JSON metadata from PyPI:

```
thoth-license-solver --package-name requests --package-version 2.26.0
```

**Describe alternatives you've considered**

Run thoth-solver and pass results to thoth-license-solver, but that might be not that intuitive (requires knowledge of thoth-solver). Also community adoption might be better if the tool provides a way to easily extract the license by passing package name and version (can default to latest).

### LINKED ISSUE #32: Save output to file
**Is your feature request related to a problem? Please describe.**

As a user of Thoth's license solver, I would like to have the ability to save license-solver output to file

```
thoth-license-solver --package-name requests --package-version  2.26.0 --output result.json
```

\+ pretty printing

### PR DESCRIPTION
**Pull request by meile18**
## Related Issues and Dependencies

Closes #27 
Closes #32

## This introduces a breaking change

- [x] No

## This Pull Request implements

- readme
- #27 
- #32 
- updated tests
- added new arguments
- fixed some bugs

### CODE REVIEW

**Comment by meile18** [comment]
@fridex can you look at it if you have time? Since it is a big update, I can explain it to you in a video call :smile:

File: thoth/license_solver/cli.py [hunk 1]
```diff
@@ -67,23 +187,49 @@ def cli(_: click.Context, directory: str, file: str, verbose: bool = False) -> N
...
 
+    # file argument
     if file:
-        _LOGGER.debug("Parsing file: %s", file)
-        license_solver.solve_from_file(file)
-        license_solver.print_output()
+        for f in file:
+            if not os.path.isfile(f):
+                _LOGGER.warning("Not a valid file %r", f)
+                print(f'Not a valid file "{f}" [SKIPPED]')
```
Comment (fridex) [inline thoth/license_solver/cli.py:222]: I'm unsure about these `print` statements. If the logger is setup properly, they should not be needed and users can adjust verbosity based on their requirements. Do you use `init_logging` from thoth-common?

File: README.md [hunk 2]
```diff
@@ -1,36 +1,175 @@
...
+        }
+    }
 
-It is a good habit for the program to be tested after the implementation of new features. You can run:
+    ```
 
+## Installation
+Install `license-solver`:
+```shell
+$ git clone https://github.com/thoth-station/license-solver     # before clone, select folder where to clone repository
```
Comment (fridex) [inline README.md:164]: The default installation method could be from PyPI (once the package is published):

```
pip install thoth-license-solver
```

The git-clone method could be for development and testing the tool.

**Review by fridex — COMMENTED** [review]
> @fridex can you look at it if you have time? Since it is a big update, I can explain it to you in a video call smile

Few very minor comments left, otherwise, the code looks good to me. Let's discuss details on the call. 🤙🏻 👍🏻

**Comment by meile18** [comment]
@fridex if you have time can you review pls. All your review comments were fixed.

**Review by fridex — APPROVED** [review]
Looks really good to me. I'm not that familiar with the codebase so cannot judge on changes. Let's have this in. Nice job! 💯 👍🏻

**Pull request merged**